spec/support/database_cleaner.rb

```diff
@@ -1,6 +1,16 @@
 RSpec.configure do |config|
-  config.before(:each) do
+  config.before(:suite) do
     DatabaseCleaner.clean_with(:truncation)
```

Comment:
But I think that should always remain true unless someone wants to manually run the factory_girl rake task which indeed won't be possible as one of our factory build depend on internet.